packages/components/src/List/Toolbar/ColumnChooserButton/ColumnChooserButton.component.js

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { Overlay, Popover } from 'react-bootstrap';
+import ColumnChooser from './ColumnChooser';
+import getDefaultT from '../../../translate';
+import ActionButton from '../../../Actions/ActionButton';
+
+export default function ColumnChooserButton({
+	children,
+	columns,
+	filterValue,
+	id,
+	initialOpenedPopover,
+	placement = 'left',
+	nbLockedLeftItems,
+	submit,
+	t = getDefaultT(),
+}) {
+	const [opened, setOpened] = useState(initialOpenedPopover || false);
+	const [buttonRef, setButtonRef] = useState(null);
```

Comment:
Use useRef for references

Reply:
Try to change it and you will have a nice surprise.